operations/github/onboarding/repo_init_metadata/security_baseline.json

```diff
@@ -8,7 +8,7 @@
   },
   "main_team": {
       "name": "sig-security-baseline",
-      "parent_team": "test",
+      "parent_team": "SIGs",
       "privacy": "closed",
       "notification_setting": "true",
       "permission": "pull",
```

Comment:
Is this a valid permission?  I looks like you're using this in the call to this https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#add-or-update-team-project-permissions

That lists only "read, write, admin" as the possible values.  I think the "pull" value is deprecated.  I would assume that this should allow for the "maintain" role, but it isn't in the documentation.

Reply:
this is the API that the code uses to add or update repo access
https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#add-or-update-team-repository-permissions